CHANGES.md

```diff
@@ -12,6 +12,7 @@
         `markdown-disable-tooltip-prompt`.
     -   Introduce `markdown-ordered-list-enumeration` variable [GH-587][]
     -   Search wiki link under project
+    -   Add `markdown-insert-gfm-foldable-block` function [GH-596][]
```

Comment:
Missed while reviewing the first time: I guess it shouldn't have `gfm` since you renamed it ?

Reply:
sorry I'll fix it later

Reply:
https://github.com/jrblevin/markdown-mode/commit/d8267c17eb14d3db2ebd079094df524b547607b1

Reply:
No worries. :-) Thanks for adding the test case and making the code better!